com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -1182,6 +1182,45 @@ def __init__(self, rest_client, credentials):
     def _get_url(self, req_name):
         return self._credentials['rest_url'] + self._credentials[req_name]
 
+    def submit_job(self, sab_file, application_name,  parameters=[], configuration={}, tracing='info' ):
+        """Submit a compiled Streams Application Bundle (a SAB file) to run in this Streamin Analytics service.
+        
+        Args:
+        sab_file (string): path to the compiled SAB file containing the application to be submitted
+        application_name (string): name of the application in the SAB file
+        parameters (list of {name/value} pairs, optional): application's submission-time parameters
+        configuration (dict, optional): hash of deployment configuration parameters 
+        tracing (string, optional): one of 'off', 'error', 'warn', 'info',' debug', or 'trace', defaults to 'info'
```

Comment:
Elsewhere we use Python tracing levels to indicate the level, can it match the existing use.

Also the default should be `None` to use the instance default.

Reply:
Agreed -- I've updated the submit_job() method to pass 'tracing' via an optional job config overlay.